### DISCUSSION TITLE
Search by episode name

### DISCUSSION THREAD

**Discussion by int0x21** [pymedusa/Medusa#4513] [Ideas & Feature requests]
A feature request i assume eveyone who downloads documentaries would love is if the searcher could be set to search for epsiode name as many documentaries (for example bbc documentaries) are named like this

**Comment by p0psicles** [comment]
Already in the works

**Comment by ericvlog** [comment]
it already done?
thanks regard.

**Comment by ericvlog** [comment]
because mvgroup.org only adding torrent by it episode name.
mvgroup got massive documentary content.
